x-pack/platform/plugins/shared/alerting/server/routes/rule/apis/get/transforms/transform_get_response/v1.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { transformRuleToRuleResponseV1 } from '../../../../transforms';
+import type { Rule, RuleParams } from '../../../../../../application/rule/types';
+
+export const transformGetResponse = <Params extends RuleParams>(
+  rule: Rule<Params>,
+  includeArtifacts: boolean = true
```

Comment:
I used as default value true based on the default state [here](https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/alerting/server/routes/rule/apis/get/get_rule_route.ts#L36) (internal route)

Reply:
I defaulted to false in this [commit](https://github.com/elastic/kibana/pull/219462/commits/a752a25d83593f6e18e75f7d0b8ae110f311a8ca)